packages/docmanager-extension/src/index.ts

```diff
@@ -784,7 +784,7 @@ function addLabCommands(
 
   commands.addCommand(CommandIDs.rename, {
     label: () =>
-      trans.__('Rename %1…', fileType(contextMenuWidget(), docManager)),
+      trans.__('Rename%1…', ' ' + fileType(contextMenuWidget(), docManager)),
```

Comment:
Oh wait, I guess my suggested change always puts the space in there anyway. We really only want the space there if that fileType method returns a nonempty string. How about something like this, which only adds the space if needed?

label: () => {
  const t = fileType(contextMenuWidget(), docManager));
  if (t) {
    t = ' ' + t;
  }
  return trans.__('Rename%1…', t)
},

Reply:
Thanks I included this :) I just had to change const to let in order for it to compile properly